rules/scripts/license_check.bash.template

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
It looks like the license checker doesn't know about .bash.template files

Reply:
It looks like the license checker only checks files with particular extensions, and these extensions are coded into the program rather than listed in a configuration file.  The license checker is "vendored" code, so I'm not going to change it; rather I renamed these files as `foo.template.sh`.

Similarly, I eliminated the `BUILD` file from the license-checker subdir so the current CI's vendoring check won't complain.